### DISCUSSION TITLE
CIRCUT option

### DISCUSSION THREAD

**Discussion by devdems** [serkri/SmartEVSE-3#250] [General]
Hi,

i'm having a hard time understanding the CIRCUT parameter.

It is written like this:

CIRCUIT	(only appears when PWR SHARE set to <Master>, or when PWR SHARE set to <Disabled>
        and Mode is Smart or Solar):
        Set the max current the EVSE circuit can handle (load balancing): 10-200A
        (see also subpanel wiring)

Why would this option be available to you if you have PWR SHARE set to Disabled meaning you only have one SmartEvse? We already have the MAX option that takes care of the max Amps allowed or I'm missing something?

**Comment by dingo35** [comment]
If you have 1 EVSE you can still have other devices connected to your subpanel. If you want to guard the subpanel currents, put the EVMeter there and the Circuit parameter will limit the currents so that EVMeter will never exceed Circuit.
See also the hardware schematics in the docs.

This is also called "garage config" since many users have there EVSE in the garage, where there are also other current uaers.

**Comment by devdems** [comment]
@dingo35 i see what you mean but wouldn't that be possible only if you have EVMeter so it can check the consumption on that specific subpanel? I for example only have mains meter.

So wouldn't the condition to show CIRCUT need to be (only appears when PWR SHARE set to Master, or when PWR SHARE set to Disabled and Mode is Smart or Solar **AND if EV METER not set to Disabled**):

**Comment by dingo35** [comment]
Yeah we have some feature requests that are much more exciting then this one...

**Comment by devdems** [comment]
Agreed. All good, just so I'm clear how this is used. Tnx for clearing this out for me.
